src/ddqa/models/github.py

```diff
@@ -17,6 +17,8 @@ class PullRequestReviewer(BaseModel):
 
 
 class TestCandidate(BaseModel):
+    __test__ = False
+
```

Comment:
I think I'd prefer something like renaming the class when importing it on the test file, because that's closer to the problem we're trying to "solve"; or any solution that would be contained within test files and not the app's source code. The next best alternative would perhaps be to add a brief comment here to let the future reader of the code know why we need this.

Reply:
Good idea, let's rename it. 👍